index.html

```diff
@@ -46,5 +46,142 @@ <h4 class="welcome">LET'S CONNECT</h4>
       </ul>
     </section>
   </main>
+  <section class="card">
+    <figure class="picture">
+      <img src="images/Portf1.png" alt="">
+    </figure>
+    <div class="works">
+      <h2>Tonic</h2>
+      <ul class="cover">
+        <li>Canopy</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <article>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </article>
+      <ul class="buttn">
+        <li class="lngbtn">html</li>
+        <li class="lngbtn">css</li>
+        <li class="lngbtn">javaScript</li>
+      </ul>
+      <a href="" class="prLnk">See Project</a>
+    </div>
+  </section>
+  <section class="card">
+    <figure class="picture">
+      <img src="images/Portf2.png" alt="">
+    </figure>
+    <div class="works">
+      <h2>Multi-Post Stories</h2>
+      <ul class="cover">
+        <li>Canopy</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <article>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </article>
+      <ul class="buttn">
+        <li class="lngbtn">html</li>
+        <li class="lngbtn">css</li>
+        <li class="lngbtn">javaScript</li>
+      </ul>
+      <a href="" class="prLnk">See Project</a>
+    </div>
+  </section>
+  <section class="card">
+    <figure class="picture">
+      <img src="images/Portf3.png" alt="">
+    </figure>
+    <div class="works">
+      <h2>Tonic</h2>
+      <ul class="cover">
+        <li>Canopy</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+      </ul>
+      <article>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </article>
+      <ul class="buttn">
+        <li class="lngbtn">html</li>
+        <li class="lngbtn">css</li>
+        <li class="lngbtn">javaScript</li>
+      </ul>
+      <a href="" class="prLnk">See Project</a>
+    </div>
+  </section>
+  <section class="card">
+    <figure class="picture">
+      <img src="images/Portf4.png" alt="">
+    </figure>
+    <div class="works">
```

Comment:
- [ ] The issue here has been partially resolved but while solving this you introduced a bit of overflow with your images. Kindly fix that.
![FireShot Capture 102 - {a href=__}GintsMisins Portfolio{_a} - raw githack com](https://user-images.githubusercontent.com/46851023/133908163-ae9766bf-d66b-411e-b3bd-3d1314410cd5.png)

Reply:
> 
> 
>     * [ ]  The issue here has been partially resolved but while solving this you introduced a bit of overflow with your images. Kindly fix that.
>       ![FireShot Capture 102 - {a href=__}GintsMisins Portfolio{_a} - raw githack com](https://user-images.githubusercontent.com/46851023/133908163-ae9766bf-d66b-411e-b3bd-3d1314410cd5.png)

Maybe You can give me some advice How best to solve it because I was working on Firefox and adjusted, like on galaxy s9,
I know they should be responsive, but at this moment I probably lost my track already. in Readme file I added a screenshot how it looks in my.